content/logs/index.md

```diff
@@ -147,22 +194,22 @@ However, if your logs defines one of the following attributes Datadog considers
 * `eventTime`
 * `published_date`
 
-You can also ask Datadog to consider any attribute as the official date reference thanks to the [log date remapper processor](/logs/processing/#log-date-remapper)
+You can also ask Datadog to interpret the value of any other attribute as the log’s official date if you set a [log date remapper processor](/logs/processing/#log-date-remapper)
```

Comment:
You can also specify alternate attributes to use as the source of a log's date  by setting a [log date remapper processor](/logs/processing/#log-date-remapper)"

Reply:
👍